server/tests/fixtures/random_objects.py

```diff
@@ -1301,3 +1302,10 @@ async def create_advertisement_campaign(
     )
     await save_fixture(advertisement_campaign)
     return advertisement_campaign
+
+
+@pytest.fixture
+def event_creation_time() -> tuple[datetime, int]:
```

Comment:
May we put this fixture directly in `order/test_service.py`? Seems only useful there for now.

Reply:
Done ✅